### PR TITLE
update volunteer button

### PR DESCRIPTION
**Pull request by estherrose046**
### What
Changed the "seeking volunteers" button to appear in the presence of the string "yes" rather than any truthy value. 

### Why
Removes incorrect "needs volunteers" buttons. 

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [x] Tapping on a marker on the map displays information about the marker in a popup.
- [x] Tapping the "Show list of locations" button replaces the map view with a list view.
- [x] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [x] Tapping one of the ✅'s in the legend filters items on the map and in the list.

### Check the app in the following web browsers:
- [x] Chrome
- [x] Firefox
- [ ] Edge
- [x] Safari

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -122,7 +122,10 @@ const getStatus = id => _.find(statusOptions, s => (s.id === id.toLowerCase()))
 const createListItem = (location, status, lng, lat) => {
   const urgentNeed = location.urgentNeed ? `<p class="urgentNeed p location-list--important">Urgent Need: ${location.urgentNeed}</p>` : ''
   const seekingMoney = location.seekingMoney ? `<span class="seekingMoney location-list--badge">Needs Money Donations</span>` : ''
-  const seekingVolunteers = location.seekingVolunteers ? `<span class="seekingVolunteers location-list--badge">Needs Volunteer Support</span>` : ''
+  let seekingVolunteers = ''
+  if (location.seekingVolunteers && location.seekingVolunteers.match(/(?:\byes\b)/i)) {
```
Comment (jdalt) [inline script.js:126]: I would've been lazy and tried to just detect "yes" in a string but you were adamant that "yesterday" not trigger the button.

**Comment by jdalt** [comment]
Looks good:
Twin_Cities_Aid_Distribution_Locations_and_Twin_Cities_Aid_Distribution_Locations

**Comment by jdalt** [comment]
We could still do a lot to improve the experience of this button. Volunteering often has a lot of caveats.

**Comment by experimatt** [comment]
@estherrose046 and @jdalt reviewing each other's PRs feels like a next-level relationship milestone. You two are amazing! 🙌

**Pull request merged**